### PR TITLE
Admin controller

### PR DESCRIPTION
**Pull request by hotuanhung**
## WHAT (optional)
- Check logic admin page, logout admin
- Fix header transaction
- Thêm Page Admin khi là admin

## Evidence (Screenshot or Video)

Screenshot from 2023-09-18 10-20-49
Screenshot from 2023-09-18 10-52-06

Screenshot from 2023-09-17 12-01-13
Screenshot from 2023-09-17 12-05-00
Screenshot from 2023-09-17 12-29-21
Screenshot from 2023-09-18 10-54-10

### CODE REVIEW

**Comment by hotuanhung** [comment]
ready

**Comment by hotuanhung** [comment]
ready

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -13,6 +13,11 @@
          <li>
             <%= link_to t("favorites"),user_like_books_path(current_user), class: "nav-link"%>
          </li>
+         <% if current_user.admin? %> 
+            <li>
+               <%= link_to t("admin.page"), admin_static_pages_path, class: "nav-link"%>
+            </li>
+            <% end %>
```
Comment (ngocvt-0484) [inline app/views/layouts/_header.html.erb:20]: sai indent

**Comment by ngocvt-0484** [comment]
c merge tí fix ở PR sau

**Pull request merged**